database/generateData.js

```diff
@@ -0,0 +1,25 @@
+
```

Comment:
I would suggest being consistent in spacing at the top of your files, I think the Airbnb style guide wants code to start on the first line?